### ISSUE TITLE
replace react-router-redux

### ISSUE THREAD

**Issue by saghul** [jitsi/jitsi-meet-electron#59]
Looks like the project is now deprecated: https://github.com/ReactTraining/react-router/tree/master/packages/react-router-redux

> This project is no longer maintained. For your Redux <-> Router syncing needs, please see one of these libraries instead: connected-react-router

So I guess we should update to https://github.com/supasate/connected-react-router

**Comment by AkshitKrNagpal** [comment]
@saghul 
I am actually doubting should we do this.
I was much more comfortable using alpha version of [react-router-redux](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-redux) because it was part of official [react-router](https://github.com/ReactTraining/react-router) which is maintained by [ReactTraining](https://github.com/ReactTraining). 

[Connected React Router](https://github.com/supasate/connected-react-router) wasn't being maintained for a long time and has one maintainer only who said this 13 days ago in this [conversation](https://github.com/supasate/connected-react-router/issues/74) 
> I'll be back to start maintaining it again. But, my time is quite limited, so, if anyone is interested to help maintain the project, please let me know!

I think we should wait until the project picks up some speed. We are not lacking any functionality and [Connected React Router](https://github.com/supasate/connected-react-router) doesn't give any additional features.

**Comment by saghul** [comment]
Agreed. I created the issue so we can keep track of it, but we don’t need to act on it just yet.

TBH, we can go back to the DOM router, which is hopefully not deprecated day after tomorrow  🤦‍♂️
